src/translations/screens/Profile.ts

```diff
@@ -41,9 +41,17 @@ const ProfileTexts = {
       linkItems: {
         places: {
           label: _('Steder', 'Locations'),
+          a11yLabel: _(
```

Comment:
Her er jeg litt usikker, siden man på en måte pakker inn både label og aktiveringshint i samme tekst. Accesssibility tekster er ikke gjort på denne måten andre steder i appen. I dette tilfellet ville jeg droppet a11yLabel, men heller lagt på a11yHint "Aktivér for å endre favorittsteder". Da blir det et mer gjennomført mønster både i appen for oss utviklere, og også for brukerne siden label og hint leses i litt forskjellig kontekst.